codebuild/CanaryWrapper_Classes.py

```diff
@@ -840,8 +838,9 @@ def monitor_loop_function(self, psutil_process : psutil.Process, time_passed=30)
                     self.data_snapshot.post_metrics(psutil_process)
                 except Exception as e:
                     self.print_message("[SnaptshotMonitor] ERROR - exception occurred posting metrics!")
```

Comment:
Trivial: Is there any additional information we can extract from the exception that we can share in the logging?

Reply:
We could print the exception, since the logs will only ever be visible in Codebuild, there shouldn't be any credential leaking risks since only the team can see the Codebuild output.